<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepairTicketSonar</class>
 <widget class="QMainWindow" name="RepairTicketSonar">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>936</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sonar Repair Ticket Creation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QTextEdit" name="ticketTemplate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>810</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-39</y>
         <width>351</width>
         <height>847</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="ticketInfo">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Ticket Information:           &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="tktType_label">
          <property name="text">
           <string>Fix Ticket Type:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QComboBox" name="tktType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Radio down</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>No connection</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Intermittent Connection</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Slow/Interm Speeds</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Poor Signal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Conversion</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Onsite</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Misc.</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="tktZone_label">
          <property name="text">
           <string>Zone:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QComboBox" name="tktZone">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="tktTower_label">
          <property name="text">
           <string>Closest Tower:</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QComboBox" name="tktTower">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="25" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#aa0000;&quot;&gt;IMPORTANT:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="27" column="0">
         <widget class="QCheckBox" name="tktQConfirm">
          <property name="text">
           <string>Confirmation call was explained.</string>
          </property>
         </widget>
        </item>
        <item row="28" column="0">
         <widget class="QCheckBox" name="tktQAbsent">
          <property name="text">
           <string>$25 absent fee explained.</string>
          </property>
         </widget>
        </item>
        <item row="29" column="0">
         <widget class="QCheckBox" name="tktQTrip">
          <property name="text">
           <string>$65 trip fee explained.</string>
          </property>
         </widget>
        </item>
        <item row="30" column="0">
         <widget class="QCheckBox" name="tktQDueDate">
          <property name="text">
           <string>Due date is correct in Sonar ticket.</string>
          </property>
         </widget>
        </item>
        <item row="31" column="0">
         <widget class="QLabel" name="tktQDisclose_label">
          <property name="text">
           <string>Any animals onsite must be contained while our technician is on site.</string>
          </property>
         </widget>
        </item>
        <item row="32" column="0">
         <widget class="QCheckBox" name="tktQDisclose">
          <property name="text">
           <string>Disclosure Read.</string>
          </property>
         </widget>
        </item>
        <item row="33" column="0">
         <widget class="QLabel" name="tktNotes_label">
          <property name="text">
           <string>Notes / Problem:</string>
          </property>
         </widget>
        </item>
        <item row="34" column="0">
         <widget class="QTextEdit" name="tktNotes">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>What was the problem that the customer was reporting?What was done on the call? Include the customer's attitude. Were they tech savy or not?</string>
          </property>
         </widget>
        </item>
        <item row="36" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Customer Information:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="35" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="38" column="0">
         <widget class="QLineEdit" name="cstName"/>
        </item>
        <item row="37" column="0">
         <widget class="QLabel" name="cstName_label">
          <property name="text">
           <string>Customer Name:</string>
          </property>
         </widget>
        </item>
        <item row="40" column="0">
         <widget class="QLineEdit" name="cstTest"/>
        </item>
        <item row="39" column="0">
         <widget class="QLabel" name="cstTest_label">
          <property name="text">
           <string>Customer Speed Test Results:</string>
          </property>
         </widget>
        </item>
        <item row="42" column="0">
         <widget class="QLineEdit" name="cstTorch"/>
        </item>
        <item row="41" column="0">
         <widget class="QLabel" name="cstTorch_label">
          <property name="text">
           <string>Torch Results:</string>
          </property>
         </widget>
        </item>
        <item row="43" column="0">
         <widget class="QGroupBox" name="cst_Group_Package">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>125</height>
           </size>
          </property>
          <property name="title">
           <string>Service Package:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="cstPackRes">
             <property name="text">
              <string>Residential</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="cstPackBus">
             <property name="text">
              <string>Business</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="cstPackOth">
             <property name="text">
              <string>Other</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cstPackage">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
       <zorder>ticketInfo</zorder>
       <zorder>tktType</zorder>
       <zorder>tktZone</zorder>
       <zorder>tktType_label</zorder>
       <zorder>tktZone_label</zorder>
       <zorder>tktTower</zorder>
       <zorder>tktTower_label</zorder>
       <zorder>tktQAbsent</zorder>
       <zorder>tktQTrip</zorder>
       <zorder>label_11</zorder>
       <zorder>tktQConfirm</zorder>
       <zorder>tktQDisclose</zorder>
       <zorder>tktQDisclose_label</zorder>
       <zorder>tktNotes_label</zorder>
       <zorder>tktNotes</zorder>
       <zorder>verticalSpacer_3</zorder>
       <zorder>tktQDueDate</zorder>
       <zorder>label_4</zorder>
       <zorder>verticalSpacer_2</zorder>
       <zorder>cstName</zorder>
       <zorder>cstName_label</zorder>
       <zorder>cstTest</zorder>
       <zorder>cstTest_label</zorder>
       <zorder>cstTorch</zorder>
       <zorder>cstTorch_label</zorder>
       <zorder>cst_Group_Package</zorder>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>810</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-27</y>
         <width>351</width>
         <height>835</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="19" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Radio Information:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="23" column="0">
         <widget class="QLabel" name="radioIP_label">
          <property name="text">
           <string>Manage IP:</string>
          </property>
         </widget>
        </item>
        <item row="24" column="0">
         <widget class="QLineEdit" name="radioIP">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <widget class="QLabel" name="radioPublic_label">
          <property name="text">
           <string>Public IP:</string>
          </property>
         </widget>
        </item>
        <item row="26" column="0">
         <widget class="QLineEdit" name="radioPublic">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="27" column="0">
         <widget class="QLabel" name="radioMAC_label">
          <property name="text">
           <string>Radio MAC Address:</string>
          </property>
         </widget>
        </item>
        <item row="28" column="0">
         <widget class="QLineEdit" name="radioMAC">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="30" column="0">
         <widget class="QLineEdit" name="radioTest"/>
        </item>
        <item row="32" column="0">
         <widget class="QCheckBox" name="radioFirmware">
          <property name="text">
           <string>Check the Radio's Firmware.</string>
          </property>
         </widget>
        </item>
        <item row="37" column="0">
         <widget class="QGroupBox" name="radio_Group_Type">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>70</height>
           </size>
          </property>
          <property name="title">
           <string>Radio Type:</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="radioType">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UBNT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ePMP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Canopy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LTE - Telrad</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="radioTypeType">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="38" column="0">
         <widget class="QLabel" name="radioSSID_label">
          <property name="text">
           <string>AP SSID:</string>
          </property>
         </widget>
        </item>
        <item row="39" column="0">
         <widget class="QLineEdit" name="radioSSID"/>
        </item>
        <item row="40" column="0">
         <widget class="QLabel" name="radioQAPCount_label">
          <property name="text">
           <string>AP Customer Count:</string>
          </property>
         </widget>
        </item>
        <item row="41" column="0">
         <widget class="QLineEdit" name="radioQAPCount"/>
        </item>
        <item row="45" column="0">
         <widget class="QLabel" name="radioCCQ_label">
          <property name="text">
           <string>CCQ:</string>
          </property>
         </widget>
        </item>
        <item row="46" column="0">
         <widget class="QLineEdit" name="radioCCQ"/>
        </item>
        <item row="47" column="0">
         <widget class="QLabel" name="radioQuaCap_label">
          <property name="text">
           <string>Quality / Capacity:</string>
          </property>
         </widget>
        </item>
        <item row="48" column="0">
         <widget class="QLineEdit" name="radioQuaCap"/>
        </item>
        <item row="49" column="0">
         <widget class="QLabel" name="radioSignal_label">
          <property name="text">
           <string>Signal Strength:</string>
          </property>
         </widget>
        </item>
        <item row="50" column="0">
         <widget class="QLineEdit" name="radioSignal"/>
        </item>
        <item row="51" column="0">
         <widget class="QLabel" name="radioLastSignal_label">
          <property name="text">
           <string>Last Known Good Signal:</string>
          </property>
         </widget>
        </item>
        <item row="63" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="52" column="0">
         <widget class="QLineEdit" name="radioLastSignal">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="29" column="0">
         <widget class="QLabel" name="radioTest_label">
          <property name="text">
           <string>Radio Speed Test:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="Copy">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Copy to Clipboard</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Generate Ticket</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="Clear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>tktType</tabstop>
  <tabstop>tktZone</tabstop>
  <tabstop>tktTower</tabstop>
  <tabstop>tktQConfirm</tabstop>
  <tabstop>tktQAbsent</tabstop>
  <tabstop>tktQTrip</tabstop>
  <tabstop>tktQDueDate</tabstop>
  <tabstop>tktQDisclose</tabstop>
  <tabstop>tktNotes</tabstop>
  <tabstop>cstName</tabstop>
  <tabstop>cstTest</tabstop>
  <tabstop>cstTorch</tabstop>
  <tabstop>radioIP</tabstop>
  <tabstop>radioPublic</tabstop>
  <tabstop>radioMAC</tabstop>
  <tabstop>radioTest</tabstop>
  <tabstop>radioFirmware</tabstop>
  <tabstop>radioType</tabstop>
  <tabstop>radioTypeType</tabstop>
  <tabstop>radioSSID</tabstop>
  <tabstop>radioQAPCount</tabstop>
  <tabstop>radioCCQ</tabstop>
  <tabstop>radioQuaCap</tabstop>
  <tabstop>radioSignal</tabstop>
  <tabstop>radioLastSignal</tabstop>
  <tabstop>ticketTemplate</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Copy</tabstop>
  <tabstop>scrollArea_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
